packages/delisp-core/package.json

```diff
@@ -4,6 +4,7 @@
   "main": "dist/src/index.js",
   "license": "MIT",
   "scripts": {
+    "tsn": "ts-node",
```

Comment:
Should we install this this toplevel maybe?

Reply:
Doesn't work unless you specify the `--project` manually, which I tried to avoid this way.